Plugins/NativeServerPlugin/src/directx_buffer_capturer.cpp

```diff
@@ -42,6 +42,13 @@ void DirectXBufferCapturer::SendFrame(int64_t prediction_time_stamp)
 {
 	if (!headless_)
 	{
+		MessageBox(
```

Comment:
can we use c++ exceptions instead of this? showing a popup as a result of this error is a decision that should be made by a higher level component. 

i'd greatly prefer `throw std::exception("Headless mode hasn't been initialized")` and __optionally__ a catch at the callsite that decides to show a messagebox.

Reply:
Hmm, there are multiple calls of **MessageBox** in *Conductor* class so this one looks consistent to me. What do you think?